src/include/storage/copier/copy_node_npy.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
```

Comment:
I am not involved in the discussion of npy design.
Should we let CopyNodeNpy inherit the base copyStructuresArrow?

Reply:
Refactored and `CopyNodeNpy` now inherits `CopyNodeArrow`